sensor/kubernetes/certrefresh/localscanner_tls_issuer.go

```diff
@@ -112,32 +112,33 @@ func (i *localScannerTLSIssuerImpl) Capabilities() []centralsensor.SensorCapabil
 	return []centralsensor.SensorCapability{centralsensor.LocalScannerCredentialsRefresh}
 }
 
-// ResponsesC is called "responses" because for other SensorComponent it is central that
-// initiates the interaction. However, here it is sensor which sends a request to central.
+// ResponsesC is called "responses" because for other SensorComponent it is Central that
+// initiates the interaction. However, here it is Sensor which sends a request to Central.
 func (i *localScannerTLSIssuerImpl) ResponsesC() <-chan *message.ExpiringMessage {
 	return i.msgToCentralC
 }
 
-// ProcessMessage dispatches Central's messages to Sensor received via the central receiver.
+// ProcessMessage dispatches Central's messages to Sensor received via the Central receiver.
 // This method must not block as it would prevent centralReceiverImpl from sending messages
 // to other SensorComponents.
 func (i *localScannerTLSIssuerImpl) ProcessMessage(msg *central.MsgToSensor) error {
-	switch m := msg.GetMsg().(type) {
-	case *central.MsgToSensor_IssueLocalScannerCertsResponse:
-		response := m.IssueLocalScannerCertsResponse
+	if m, ok := msg.GetMsg().(*central.MsgToSensor_IssueLocalScannerCertsResponse); ok {
```

Comment:
Hmm, I guess this works, but we usually do this a bit differently. Here is a pseudocode that I see in other places:

```go
response := msg.GetIssueLocalScannerCertsResponse()
if response == nil {
	return nil
}
// rest of the logic
```

This also uses the inverted if-logic (return if the message is of wrong type) and this saves us one level of indent.